.typos.toml

```diff
@@ -0,0 +1,6 @@
+[default]
+extend-ignore-identifiers-re = [
+    "alle",
+    "groupe",
+    "UE1",
```

Comment:
This pattern appears as a testing placeholder sometimes!

https://github.com/slackapi/slack-cli/blob/f3ba82430864d4434620c973bd9d089c19b1e979/internal/prompts/app_select_test.go#L4042